packages/exchange-connector/gulpfile.ts

```diff
@@ -19,12 +19,33 @@ import {
   getSpecificApi,
   groupByCategory
 } from "./src";
+import { sdkLogger } from "../core";
 
 require("dotenv").config();
 
 import config from "../../build-config";
 import { removeRamlLinks } from "./src/exchangeTools";
 
+/**
+ * Remove all the information that could be invalid so that the SDK is not
+ * generated with incorrect information accidentally. Correct information
+ * needs to be filled manually in 'apis/api-config.json' before the SDK can be
+ * generated.
+ *
+ * @param api The API with incorrect information
+ */
+function removeInvalidInformation(api: RestApi) {
```

Comment:
The method should not be in the gulpfile.  Gulpfiles should be as thin as possible.